### PR TITLE
Add HowGay with Updated Features

### PR DESCRIPTION
**Pull request by mangowhite**
msedge_YWUVE2wviX

**What this PR does and why it should be merged**

• How Gay displays a top.
• How Gay has an 100% Gay Easter-egg.
• How Gay is easy to use.
• How Gay saves the values in the database.

**Status**

* [X] Code has been tested on an official instance of YAGPDB, or there are no code changes
* [X] I've read and understood the [contribution guidelines](../CONTRIBUTING.md)

### CODE REVIEW

File: howgay.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+EDITTED BY: mangowhite (209208777365389312)
+SUGGESTED TRIGGER: howgay
+
+DESCRIPTION: This CC gives the user an amount of Gayness that will attach to them like a parasite that will never go away. But that's not all... it will also display all the most gays in the server in a Top. 
+
+/*}}
+
+{{/*VARIABLES*/}}
+
+{{$howGay:=randInt 0 101}}
```
Comment (l-zeuch) [inline howgay.go:13]: can just be `... randInt 101`

File: howgay.go [hunk 2]
```diff
@@ -0,0 +1,48 @@
+{{/*
+
+MADE BY: DZ (438789314101379072)
+EDITTED BY: mangowhite (209208777365389312)
+SUGGESTED TRIGGER: howgay
+
+DESCRIPTION: This CC gives the user an amount of Gayness that will attach to them like a parasite that will never go away. But that's not all... it will also display all the most gays in the server in a Top. 
+
+/*}}
```
Comment (l-zeuch) [inline howgay.go:9]: doesn't match requested format, refer to README

File: howgay.go [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+{{/*VARIABLES*/}}
```
Comment (l-zeuch) [inline howgay.go:11]:
```suggestion

```
this might confuse users that there is config, although there is not.

File: howgay.go [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+{{$howGay:=randInt 0 101}}
+{{$po:=0}}
+{{$color:=randInt 16777216}}
+{{$user:= ""}}{{with .Message.Mentions}}{{$user = userArg (index . 0)}}{{end}}
+{{$embed:= sdict "color" $color "title" "How Gay"}}
+
+
+{{/*CODE*/}}
```
Comment (l-zeuch) [inline howgay.go:20]: move this further up

**Review by l-zeuch — CHANGES_REQUESTED** [review]
found a few things :')

File: howgay.go (hunk 1 above)
Comment (mangowhite) [reply howgay.go]: Done!

File: howgay.go (hunk 2 above)
Comment (mangowhite) [reply howgay.go]: Done!

File: howgay.go (hunk 3 above)
Comment (mangowhite) [reply howgay.go]: Removed!

**Comment by mangowhite** [comment]
All done!

File: howgay.go (hunk 4 above)
Comment (mangowhite) [reply howgay.go]: Done.

**Review by l-zeuch — APPROVED** [review]

**Pull request merged**